site/_resources/upgrading.md

```diff
@@ -54,6 +54,14 @@ You'll need to re-check where your DNS names are pointing as well, using [Get yo
 **Note:** If you deployed Contour into a different namespace than `projectcontour` with a standard example, please delete that namespace.
 Then in your editor of choice do a search and replace for `projectcontour` and replace it with your preferred name space and apply the updated manifest.
 
+**Note:** If you are deploying to a cluster where you have previously installed alpha versions of the Contour API, applying the Contour CRDs may fail with a message similar to `Invalid value: "v1alpha1": must appear in spec.versions`. In this case, you need to delete the old CRDs before applying the new ones.
+
+```
+$ kubectl delete namespace projectcontour
+$ kubectl get crd  | awk '/projectcontour.io|contour.heptio.com/{print $1}' | cut -d ' ' -f 1 | xargs kubectl delete crd
```

Comment:
You don't need both the awk and the cut, so you can remove the cut. FWIW the reason I suggested the awk is that it's more verbose and better documents to the reader what this pipeline is doing.

Reply:
oh, of course, thanks for catching that!